src/schemas/ingestion/polygonParts.schema.ts

```diff
@@ -78,10 +75,14 @@ export const polygonPartsEntityNameSchema = z
   })
   .describe('polygonPartsEntityNameSchema');
 
+export const polygonPartsFeatureSchema = featureSchema(polygonSchema, partSchema);
+
+export const polygonPartsFeatureCollectionSchema = featureCollectionSchema(polygonPartsFeatureSchema);
+
 export const polygonPartsPayloadSchema = z.object({
   productType: rasterProductTypeSchema,
   productId: resourceIdSchema,
   catalogId: z.string().uuid(),
```

Comment:
we should have a single definition and reuse it

Reply:
each single "id" field is mentioned once (for jobId, catalogId, taskId)
so i guess we should really need to reuse it.
<img width="286" height="282" alt="image" src="https://github.com/user-attachments/assets/6264f66a-4f43-42f7-88b9-47a27a0ae2c4" />